master/struct.html

```diff
@@ -2302,7 +2301,7 @@ <h3 id="roleattribute">Role attribute</h3>
         <th>Animatable</th>
       </tr>
       <tr>
-        <td><dfn id="RoleAttribute">role</dfn></td>
+        <td><dfn id="RoleAttribute" data-dfn-type="element-attr" data-dfn-export>role</dfn></td>
```

Comment:
this needs a data-dfn-for, but I've no idea how to say apply to all elements without listing every single SVG element.